### DISCUSSION TITLE
How use a common from mail id and  different ‘from name’

### DISCUSSION THREAD

**Discussion by sierratec** [PHPMailer/PHPMailer#3056] [General]
We are using phpmailer  and using the SMTP details as below. We want to use a common from mail id and need to use different ‘from name’

$mail->From="alerts@xxxx.com";
$mail->FromName = "WebNotification, xxxx";

In the mails that we received are having the ‘from name’ ‘Alerts, xxxx’ and not 'WebNotification, xxxx'

Please suggest how use a common from mail id and  different ‘from name’

Thanks

**Comment by Synchro** [comment]
The from name is not altered by PHPMailer – it may be happening on the receiving side. Set `SMTPDebug = 2` to see exactly what is being sent, and look at the source code of the received message in your mail client.

**Reply by sierratec** [reply]
I tried after enabling debug and noticed the following. 
2024-05-20 13:06:22 CLIENT -> SERVER: From: "Mantis, SIERRA" <alerts@sierratec.com>
But still 'From name' is taken as 'Alert, SIERRA'.
Thanks
